src/wp-content/themes/twentytwentytwo/inc/patterns/footer-about-title-logo.php

```diff
@@ -0,0 +1,32 @@
+<?php
+/**
+ * Footer with text, title, and logo
+ */
+return array(
+	'title'      => __( 'Footer with text, title, and logo', 'twentytwentytwo' ),
+	'categories' => array( 'twentytwentytwo-footers' ),
+	'blockTypes' => array( 'core/template-part/footer' ),
+	'content'    => '<!-- wp:group {"align":"full","style":{"spacing":{"padding":{"top":"8rem","bottom":"6rem"}}},"backgroundColor":"secondary","layout":{"inherit":true}} -->
+					<div class="wp-block-group alignfull has-secondary-background-color has-background" style="padding-top:8rem;padding-bottom:6rem"><!-- wp:columns {"align":"wide"} -->
+					<div class="wp-block-columns alignwide"><!-- wp:column {"width":"33%"} -->
+					<div class="wp-block-column" style="flex-basis:33%"><!-- wp:paragraph -->
+					<p>' . esc_html__( 'ABOUT US', 'twentytwentytwo' ) . '</p>
```

Comment:
This may not be a big deal, but I'm wondering if this text should be in normal, sentence casing, and `text-transform: uppercase;` used instead.

I am not multilingual, but I'm not sure if the capitalization is a big deal here and in other strings throughout the theme.

Reply:
I've shared this in the [#polyglots channel in Slack](https://wordpress.slack.com/archives/C02RP50LK/p1636400886178100) to get some feedback.

Reply:
I think I've read that writing the text out in all caps may make certain screen readers announce it as an acronym? Not sure how accurate that is (was)?

Reply:
@TimothyBJacobs That does ring a bell for me too. @audrasjb would you be able to confirm this? Does the CSS transform property prevent the text from being read as an acronym if so?

Reply:
cc @kjellr

Reply:
Unless there's a really good reason for it, please avoid using all uppercase. Things get harder to read. Alsp don't enforce uppercase via CSS, since quite a few languages would still translate "ABOUT US" into "Sentence case" anyway.
Also remember that a lot of languages don't even have any different uppercase letters.

Reply:
Gutenberg ships with a case setting for text blocks, so we will switch to that here.